### ISSUE TITLE
Prompt user to add a filter that displays problematic values when it's not possible to set a column constraint

### ISSUE THREAD

**Issue by seancolsen** [mathesar-foundation/mathesar#773]
When adding a column constraint, either through #772 or #771, there's a possibility that the constraint will fail because the table values do not already support it. In this case, we should display a modal dialog with a button that sets the filter clauses to show the problematic values

![image](https://user-images.githubusercontent.com/42411/139347343-d432addf-e70a-4d69-ac3f-eb9ca994e3ca.png)

## Status

- Blocked by #1028 because we need a way to filter on duplicate values
- Blocked by #1394 because the front end needs to know the reason that a NOT NULL constraint failed to be set.

**Comment by seancolsen** [comment]
@kgodey I would be inclined to move this to a later milestone. What do you think? Without this feature implemented, users will still be able to set constraints. If the action fails, the user will see a toast message explaining the reason. The feature in this ticket is definitely worth implementing, but I think it's lower-priority than some of the other features that we have have in our queue for the alpha release.

**Comment by kgodey** [comment]
@seancolsen Good point, I moved it to the "Improvements for Non-Technical Users" milestone.

**Comment by kgodey** [comment]
Closing this, too old, requirements are out of date. We can create a new issue if we need this functionality in the future.
